plugins/build-your-own.rst

```diff
@@ -25,13 +25,9 @@ This interface defines the ``handleRequest`` method that allows to modify behavi
      */
     public function handleRequest(RequestInterface $request, callable $next, callable $first);
 
-The ``$request`` comes from an upstream plugin or Plugin Client itself.
+The ``$request`` comes from an upstream plugin or ``PluginClient`` itself.
 You can replace it and pass a new version downstream if you need.
 
-.. note::
-
-    Be aware that the request is immutable.
```

Comment:
i found this mostly confusing. i can't tell what we wanted to say with this. we say that you can replace the request for a new version that you pass downstream. what else is there to say? the note made me wonder if the request would not be allowed to be changed.